server/models/Request.js

```diff
@@ -0,0 +1,42 @@
+import mongoose from ("mongoose");
+import dbConnection from "./../db/mongoose";
+
+const RequestSchema = mongoose.Schema(
+    {
+        userId: [
+            {
+                type: mongoose.Schema.Types.ObjectId,
+                ref: 'User',
+                required: true
+            }
+        ],
+        requestedUserId: [
+            {
+                type: mongoose.Schema.Types.ObjectId,
+                ref: 'User',
+                required: true
+            }
+        ],
+        startTime: {
+            type: Date,
+            required: true
+        },
+        endTime: {
+            type: Date,
+            required: true
+        },
+        accepted: {
```

Comment:
Maybe we should have this as "status". The states can be "pending", "declined", or "accepted". Let me know what you think.

Reply:
status : { type: String } 
and select value from  "pending", "declined", and "accepted",  like this? ?

Reply:
Yup looks good!